### PR TITLE
Move imports from bundle to widgets

### PR DESCRIPTION
**Pull request by babich-a**

### CODE REVIEW

File: styles/widgets/generic/validation.generic.less [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@import (once) "./overlay.generic.less";
 @import (once) "../base/validation.less";
```
Comment (BingoRUS) [inline styles/widgets/generic/validation.generic.less:1]: Maybe it's better to strengthen CSS selectors in base/validation.less and import theme-specific less after the base one?

File: styles/widgets/ios7/popup.ios7.less [hunk 2]
```diff
@@ -1,3 +1,7 @@
+@import (once) "./button.ios7.less";
+@import (once) "./overlay.ios7.less";
+// @import (once) "./toolbar.ios7.less";
```
Comment (BingoRUS) [inline styles/widgets/ios7/popup.ios7.less]: Why commented?

File: styles/widgets/generic/validation.generic.less (hunk 1 above)
Comment (babich-a) [reply styles/widgets/generic/validation.generic.less:1]: I'll change the order

File: styles/widgets/ios7/popup.ios7.less (hunk 2 above)
Comment (babich-a) [reply styles/widgets/ios7/popup.ios7.less]: Yes. My fail. I'll fix

**Review by BingoRUS — APPROVED** [review]

**Pull request merged**